src/peerstate.rs

```diff
@@ -82,13 +80,24 @@ pub struct Peerstate {
     /// Fingerprint of the verified public key.
     pub verified_key_fingerprint: Option<Fingerprint>,
 
+    /// The address that verified this verified key.
+    pub verifier: Option<String>,
```

Comment:
isn't this an introducer?  Just because someone introduces a verified key does not make them the verified of that key so i think "introducer" is better, and also matches UI.

Reply:
this PR introduces several new names and columns on the Rust-level -- didn't mean to suggest to rename existing database columns.   But if you think database/rust-code consistency is more important than UI/core-naming then at least don't say "the address that verified this key" in a new doc comment because that's not the true meaning.